streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamTaskTest.java

```diff
@@ -381,7 +381,7 @@ public void shouldRecordProcessRatio() {
 
```

Comment:
req: Please rename this unit test to reflect the new verifications.

Reply:
I will add a new function to verify the buffer size.